install.sh

```diff
@@ -474,6 +501,7 @@ OS=$(lsb_release -i | awk -F ':' '{print $2}' | awk '{print $1}')
 # major version ["deb": [12, 14, 6, 7, 8], "rpm": [6, 7, 2016]]
 MAJORVERSION=$(lsb_release -r | awk '{print $2}' | awk -F '.' '{print $1}')
 
+# TODO: 2.2.11 is out and the version we should use
```

Comment:
I grab URL's for these from the sphinxsearch download page here: http://sphinxsearch.com/downloads/release/